lib/workload/stateless/filemanager/filemanager/src/events/aws/message.rs

```diff
@@ -162,13 +162,58 @@ impl From<Message> for FlatS3EventMessages {
 mod tests {
     use serde_json::json;
 
+    use crate::events::aws::message::EventType::Created;
     use crate::events::aws::tests::{
-        assert_flat_s3_event, expected_event_bridge_record, expected_sqs_record,
-        EXPECTED_SEQUENCER_DELETED_ONE, EXPECTED_VERSION_ID,
+        assert_flat_s3_event, expected_event_bridge_record, expected_sqs_record, EXPECTED_E_TAG,
+        EXPECTED_REQUEST_ID, EXPECTED_SEQUENCER_DELETED_ONE, EXPECTED_VERSION_ID,
     };
     use crate::events::aws::EventType::Deleted;
     use crate::events::aws::FlatS3EventMessages;
 
+    #[test]
+    fn deserialize_large_size() {
+        let message = format!(
+            r#"{{
+                "version": "0",
+                "id": "2ee9cc15-d022-99ea-1fb8-1b1bac4850f9",
+                "detail-type": "Object Created",
+                "source": "aws.s3",
+                "account": "111122223333",
+                "time": "1970-01-01T00:00:00.000Z",
+                "region": "ca-central-1",
+                "resources": [
+                    "arn:aws:s3:::bucket"
+                ],
+                "detail": {{
+                    "version": "0",
+                    "bucket": {{
+                        "name": "bucket"
+                    }},
+                    "object": {{
+                        "key": "key",
+                        "size": 373588380934135341,
```

Comment:
Ahh, I see where the number comes from now... maybe we can be cheeky and alter the example so that it matches an already known constant in std? ;)

Reply:
Good call, I'll replace it with [`i64::MAX`](https://doc.rust-lang.org/std/primitive.i64.html#associatedconstant.MAX). May as well test the biggest value here.